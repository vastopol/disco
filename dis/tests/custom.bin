1101000000000000

1000000000000000

1111000000100000
1111000000100001
1111000000100010
1111000000100011
1111000000100100
1111000000100101

1100000000000000
1100000001000000
1100000010000000
1100000011000000
1100000100000000
1100000101000000
1100000110000000
1100000111000000

0000000000000000
0000001000000000
0000010000000000
0000011000000000
0000100000000000
0000101000000000
0000110000000000
0000111000000000

0010000000000000
0010001000000000
0010010000000000
0010011000000000
0010100000000000
0010101000000000
0010110000000000
0010111000000000

1010000000000000
1010001000000000
1010010000000000
1010011000000000
1010100000000000
1010101000000000
1010110000000000
1010111000000000

1110000000000000
1110001000000000
1110010000000000
1110011000000000
1110100000000000
1110101000000000
1110110000000000
1110111000000000

0011000000000000
0011001000000000
0011010000000000
0011011000000000
0011100000000000
0011101000000000
0011110000000000
0011111000000000

1011000000000000
1011001000000000
1011010000000000
1011011000000000
1011100000000000
1011101000000000
1011110000000000
1011111000000000

1001000000111111
1001001001111111
1001010010111111
1001011011111111
1001100100111111
1001101101111111
1001110110111111
1001111111111111

0110000000000000
0110001001000001
0110010010000010
0110011011000011
0110100100000100
0110101101000101
0110101101001101
0110111111111111

