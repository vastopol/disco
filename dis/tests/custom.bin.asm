ILLEGAL_OPCODE

RTI

JMP R0
JMP R1
JMP R2
JMP R3
JMP R4
JMP R5
JMP R6
JMP R7

BR 000000000
BRp 000000000
BRz 000000000
BRzp 000000000
BRn 000000000
BRnp 000000000
BRnz 000000000
BRnzp 000000000
