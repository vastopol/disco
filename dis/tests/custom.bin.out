D000

8000

C000
C040
C080
C0C0
C100
C140
C180
C1C0

0000
0200
0400
0600
0800
0A00
0C00
0E00
